tests/integration/scripts/test_validate_integration.py

```diff
@@ -171,11 +171,11 @@ def test_graph_upgrade_msg_default(flow, validate, caplog):
         },
     })
     validate(id_)
-    assert '[scheduling][dependencies][X]graph' in caplog.messages[0]
-    assert 'for X in:\n       P1Y, R1' in caplog.messages[0]
+    log_filter(caplog, contains='[scheduling][dependencies][X]graph')
+    log_filter(caplog, contains='for X in:\n       P1Y, R1')
```

Comment:
`log_filter` is just a filter, it doesn't `assert` anything, so this is now a psudo test.

Make sure to go through these tests and make sure there is an `assert` before all the `log_filter` operators.

```suggestion
    assert log_filter(caplog, contains='[scheduling][dependencies][X]graph')
    assert log_filter(caplog, contains='for X in:\n       P1Y, R1')
```